### DISCUSSION TITLE
In line refuel? Also a bot supplied fuel station?

### DISCUSSION THREAD

**Discussion by AJarOfDirt0** [Opinionated-Blueprints/10-Books-Full-of-Rails#43] [Ideas]
When using nuclear fuel, it seems like overkill to have a dedicated train to refuel the fueling station.

**Comment by cornmacabre** [comment]
I ended up manually creating an in-line refuel provider station (the "refuel train" home) which was pretty straightforward tweaking the existing templates, but I agree that it's a great default/stock provided option IMO. Loving many of the inline stations design wise.

Also the solar variants look great. The author here has done a fantastic job.

**Comment by Bocian-1** [comment]
Ah, I forgot to respond here, sorry. I'll be adding both, I was just too focused on other parts to do anything more besides fixes

**Comment by rpdelaney** [comment]
> When using nuclear fuel, it seems like overkill to have a dedicated train to refuel the fueling station.

Why? Can you say more?
